pkg/controller/grouppermission/grouppermission_controller.go

```diff
@@ -203,18 +203,23 @@ func populateCrClusterRoleNames(groupPermission *managedv1alpha1.GroupPermission
 }
 
 // populateClusterRoleBindingNames to see if ClusterRoleBinding exists in k8s ClusterRoleBindlingList
-// returns bool
-func populateClusterRoleBindingNames(clusterRoleBindingName []string, clusterRoleBindingList *v1.ClusterRoleBindingList) []string {
+// returns list of clusterRoleBinding
+func populateClusterRoleBindingNames(clusterRoleBindingNames []string, clusterRoleBindingList *v1.ClusterRoleBindingList) []string {
```

Comment:
@sam-nguyen7 I would think something like the following would be easier to read what do you think?

```
var missingRoleBindings []string
var found bool

if len(clusterRoleBindingList.Items) == len(clusterRoleBindingNames) {
	return missingRoleBindings
}

for _, crbn := range clusterRoleBindingNames {
        for _, crbl := range clusterRoleBindingList.Items {
		if crbn == crbl {
			found = true
		}
	}
	if found {
		missingRoleBindings = append(missingRoleBindings, crbn)
	}
	found = false
}

return missingRoleBindings
```

Reply:
@lisa would you think so? There might be a more elegant solution

Reply:
I would do this, except for the length check at the start. If an item is removed and another created in the list of desired objects, then the length would be equal, but the contents would be different.